contracts/Comptroller.sol

```diff
@@ -1008,6 +1008,15 @@ contract Comptroller is Ownable2StepUpgradeable, ComptrollerV1Storage, Comptroll
         return (NO_ERROR, seizeTokens);
     }
 
+    /**
+     * @notice Return all reward distributors for this pool
+     * @return Array of RewardDistributor addresses
+     */
+    function getRewardDistributors() public view returns (RewardsDistributor[] memory) {
+        RewardsDistributor[] memory _rewardsDistributors = rewardsDistributors;
+        return _rewardsDistributors;
```

Comment:
Wouldn't Solidity copy it automatically if we just `return rewardsDistributors;`?

Reply:
yeah I will change that